sys-kernel/asahi-kernel/asahi-kernel-6.3.0_p7.ebuild

```diff
@@ -36,13 +36,13 @@ SRC_URI+="
 S="${WORKDIR}/linux-${MY_P}"
 
 LICENSE="GPL-2"
-KEYWORDS="~arm64"
+KEYWORDS="arm64"
 IUSE="+edge debug"
 
 RDEPEND=""
 BDEPEND="
   edge? (
-    >=virtual/rust-1.66.1[rustfmt]
+    >=dev-lang/rust-1.69.0[rustfmt,rust-src]
```

Comment:
Missed `USE` flag that made it so that GPU accel wasn't actually working OOTB for people.